main.go

```diff
@@ -85,7 +123,18 @@ func main() {
 
 	routeHandler := api.NewRouteHandler(engine, api.NewValidator(providers))
 
+	// new default gin engine (recovery, logger middleware)
 	router := gin.Default()
+
+	// enable authentication if not dev-mode
+	if !*devMode {
+		log.Debug("enable authentication")
+		signingKey := viper.GetString("tokensigningkey")
```

Comment:
use `cfgTokenSigningKey` constant instead of literal

Reply:
thx, fixed